qutebrowser/config/configdata.yml

```diff
@@ -517,15 +518,15 @@ content.netrc_file:
     name: File
     none_ok: true
   desc: >-
-    Location of a netrc-file for HTTP authentication.
+    Netrc-file for HTTP authentication.
 
     If unset, `~/.netrc` is used.
 
 content.notifications:
   default: ask
   type: BoolAsk
   backend: QtWebKit
-  desc: Allow websites to show notifications.
+  desc: Wheter to allow websites to show notifications.
```

Comment:
Probably missed this one because of the typo? Should probably be the old version.